README.md

```diff
@@ -37,11 +39,25 @@ Before you continue, ensure you have installed the following requirements:
 
 You can now access the server at http://localhost:8080
 
-**Ports:**
+Ports: 
+
+>    By default, the server and database will be exposed on `localhost:8080` and `localhost:5432`, respectively. If you have a port conflict with either one, change their respective docker-compose entry ports as follows:  
+> - Server from `8080:80` to `<new-port>:80`  
+> - Database from `5432:5432` to `<new-port>:5432`.
+
+#### Client
+
+1. Install all dependencies
+    ```sh
+    $ cd client
+    $ npm install --force --legacy-peer-deps
```

Comment:
Not sure if the flags should be there, this can cause dependency issues, its best to leave them as the deprecated ones to maintain comparability, i.e. its better to leave the red text.

```suggestion
    $ npm install --force --legacy-peer-deps
```

Reply:
According to Vuexy documentation, that's apparently the right way to install all dependencies. See https://pixinvent.com/demo/vuexy-react-admin-dashboard-template/documentation/docs/development/installation

Reply:
ah then ignore my comment